File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    a = ['a','b','c','d','e','f','g','h','i','j']
    b = len(str(age))
    answer = []
    for i in range(b):
        answer.insert(0, a[age%10])
        age = age//10
    return ''.join(answer)
    

File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.js
function solution(age) {
    let a = ['a','b','c','d','e','f','g','h','i','j']
    let answer = []
    let b = age.toString().length
    for (let i = 0;i<b;i++){
        answer.push(a[age%10])
        age = parseInt(age/10)
    }

    return answer.reverse().join("")
}